backend/src/main/java/gov/cdc/usds/simplereport/config/SecurityConfiguration.java

```diff
@@ -30,7 +30,11 @@
  * object-level security.
  */
 @Configuration
-@Profile("!" + BeanProfiles.NO_SECURITY) // Activate this profile to disable security
+@Profile(
+    "!"
+        + BeanProfiles.NO_SECURITY
+        + " & !"
+        + BeanProfiles.CREATE_SAMPLE_DEVICES) // Activate this profile to disable security
```

Comment:
The position of this comment is a bit confusing. Which of the two profiles are you referring to?

Reply:
Whoops, forgot to update that comment :sweat_smile: I'll make it clearer!

Reply:
Looks great, now. Thanks for the quick fix!